### PR TITLE
fix(binding-opcua): fix eslint errors & warnings #586

### PR DESCRIPTION
**Pull request by erossignon**

### CODE REVIEW

**Review by danielpeintner — APPROVED** [review]
Looks good. Thanks!

File: packages/binding-opcua/src/opcua-client.ts [hunk 1]
```diff
@@ -124,7 +136,7 @@ export default class OpcuaClient implements ProtocolClient {
     public async readResource(form: OpcuaForm): Promise<Content> {
         const url = new Url(form.href);
         const endpointUrl = `${url.protocol}//${url.host}`;
-        const method = form["opc:method"] ? form["opc:method"] : "READ";
+        // const method = form["opc:method"] ? form["opc:method"] : "READ";
```
Comment (relu91) [inline packages/binding-opcua/src/opcua-client.ts:139]: Is this not used anymore? I'd delete the line directly.

File: packages/binding-opcua/src/opcua-client.ts [hunk 2]
```diff
@@ -156,15 +168,15 @@ export default class OpcuaClient implements ProtocolClient {
...
+    public async writeResource(form: OpcuaForm, content: Content): Promise<void> {
         const body = await ProtocolHelpers.readStreamFully(content.body);
-        const payload: any = content ? JSON.parse(body.toString()) : {};
+        const payload: Record<string, unknown> = content ? JSON.parse(body.toString()) : {};
         const url = new Url(form.href);
         const endpointUrl = `${url.protocol}//${url.host}`;
-        const method = form["opc:method"] ? form["opc:method"] : "WRITE";
-        const contentType = "application/x.opcua-binary";
+        // const method = form["opc:method"] ? form["opc:method"] : "WRITE";
+        // const contentType = "application/x.opcua-binary";
```
Comment (relu91) [inline packages/binding-opcua/src/opcua-client.ts:177]: Same as above, I prefer not having code hanging around. Unless you have a good reason for it.

**Review by relu91 — CHANGES_REQUESTED** [review]
Looks good! just a minor point below.

**Pull request merged**